astropy/coordinates/sky_coordinate.py

```diff
@@ -1192,7 +1205,12 @@ def get_constellation(self, short_name=False, constellation_list='iau'):
         """
         from .funcs import get_constellation
 
-        return get_constellation(self, short_name, constellation_list)
+        # drop the velocities - not needed for a pure-position application
```

Comment:
It's actually revealing a bug hidden here: the issue is that `get_constellation` coverts to PrecessedGeocentric and the velocity machinery has some corner cases there where the velocities muck up the transform due to some unit-related issues.  I think this goes back to the frames themselves, though, so didn't want it to hold up this PR, but wanted to leave the test in since that's what revealed the bug in the first place.

At some point hopefully it can be fixed (I *think* it's a bug and not a subtle thing that's failing *correctly*), and then this could remove though.  Perhaps I should add a TODO on this (mapping to an issue to track for the future)?

Reply:
Ah, hmmm. Ok, a TODO and/or an issue would be good. If you have a code snippet showing the error, can you include in the issue? I'd like to take a look at that...